src/Types.cc

```diff
@@ -142,7 +142,7 @@ std::string JoinName(
   else if (scopeNameEndsWithDelimiter || localNameStartsWithDelimiter)
     return _scopeName + _localName;
   else
-    return _scopeName + kSdfScopeDelimiter + _localName;
+    return _scopeName + std::string(kSdfScopeDelimiter) + _localName;
```

Comment:
Ah, this is what I was talking about. We shouldn't break this type of code without a tick-tock.